packages/taquito-beacon-wallet/src/taquito-beacon-wallet.ts

```diff
@@ -182,4 +197,35 @@ export class BeaconWallet implements WalletProvider {
   async clearActiveAccount() {
     await this.client.setActiveAccount();
   }
+
+  async sign(signingRequest: { payload: string; signingType: 'operation' | 'micheline' | 'raw' }) {
```

Comment:
don't think we known enough how client.requestSignPayload() works exactly, I am not to confident that we abstract this function in BeaconWallet class.  (reference documentation how to sign Michelin where the bytes need to be padded added length and and more than prefix 05) https://tezostaquito.io/docs/signing#generating-a-signature-with-beacon-sdk